source/_docs/terminus/scripting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Terminus Manual
+subtitle: Scripting Terminus
+description: Automate your workflow with Terminus.
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+nexturl: terminus/plugins/
+previousurl: terminus/commands/
+permalink: docs/terminus/:basename/
+image: terminus-thumbLarge
+searchboost: 100
+---
+
+While Terminus makes it easy to manually manipulate the Pantheon environment from your command line, its real value is in scripting. By adding Terminus to your automated tasks, you can truly make Pantheon a part of your standardized workflow.
+
+Consider the repetitive tasks you perform using the Pantheon Dashboard:
+
+ - Can those tasks be executed by Terminus commands?
+ - Can the values required by the commands be derived programmatically?
+
+If so, you consider how you can turn the task into a script.
+
+## Authentication
+
+To streamline your use of Terminus in scripts, make sure that it's authenticated first. The recommended method is to use a [machine token](/docs/terminus/install#machine-token).
+
+## Examples
+
+ - The [Pantheon Example Terminus Auto Update Script](https://github.com/pantheon-systems/example-terminus-auto-update-script){.external} demonstrates how you can use Terminus to automate plugin and theme updating for multiple sites.
+
+
+ - The [Example WordPress Composer](https://github.com/pantheon-systems/example-wordpress-composer){.external} uses Terminus to [deploy staged changes](https://github.com/pantheon-systems/example-wordpress-composer/blob/master/.circleci/deploy}to-pantheon.sh){.external} to multidev environments.
```

Comment:
> `[deploy staged changes](https://github.com/pantheon-systems/example-wordpress-composer/blob/master/.circleci/deploy}to-pantheon.sh)`

There seems to be an incorrect character with a `}` instead of `-` in the URL. With GitHub links we should probably also link to a specific commit hash, rather than `master` in case the repository changes in the future.

> The Example WordPress Composer uses

It seems like this is missing a word. Perhaps "The Example WordPress Composer project uses". I think this is also a good opportunity to link folks to our [build tools guide](https://pantheon.io/docs/guides/build-tools/). Following that guide will allow them to deploy the example.

Reply:
> With GitHub links we should probably also link to a specific commit hash, rather than master in case the repository changes in the future.

I see how linking to that file at a specific commit will make the link evergreen. But if that file is changed to the point that it no longer reflects what the doc is describing, shouldn't the doc be updated? After all, if the methodology in the repository is updated it's probably for a good enough reason that the documentation should _also_ be updated.

Reply:
> I see how linking to that file at a specific commit will make the link evergreen

Yes, we should make the link evergreen.

> if that file is changed to the point that it no longer reflects what the doc is describing, shouldn't the doc be updated? After all, if the methodology in the repository is updated it's probably for a good enough reason that the documentation should also be updated.

Yes but I think it is better to have an evergreen link and make a PR to update the doc with a new link if the repo changes rather than having a potentially broken/confusing link in the doc.